src/Crypto/WebAuthn/Metadata/Service/Decode.hs

```diff
@@ -36,6 +40,18 @@ decodeMetadataPayload ServiceIDL.MetadataBLOBPayload {..} = do
   let mpEntries = foldMap NE.toList decodedEntries
   pure ServiceTypes.MetadataPayload {..}
 
+-- | Aeson parser for the MetadataBlob, exported for use in tests.
+metadataPayloadParser :: HashMap Text Aeson.Value -> Aeson.Parser ServiceIDL.MetadataBLOBPayload
```

Comment:
This isn't used in the tests (anymore I think). As such we can directly make this an internal function in the `Processing` module.